src/mmw/js/src/modeling/models.js

```diff
@@ -194,9 +194,9 @@ var ScenarioModel = Backbone.Model.extend({
     },
 
     initialize: function() {
-        // TODO: When fetching model or populating from a preloaded source
-        // this attribute should not be initialized as empty.
-        this.set('modifications', new ModificationsCollection());
+        if (this.get('modifications') === null || "[]") {
```

Comment:
@kdeloach Wrapping that line in a conditional fixed the issue. Everything else appears to be working as before.